server/lib/utils.js

```diff
@@ -1,3 +1,33 @@
+const moment = require('moment');
+
+const EPOCH_OFFSET = 946684800;
+module.exports.EPOCH_OFFSET = EPOCH_OFFSET;
+module.exports.XRP_BASE = 1000000;
+
+const convertRippleDate = date =>
+  moment
+    .unix(date + EPOCH_OFFSET)
+    .utc()
+    .format();
+
+module.exports.summarizeLedger = (ledger, txDetails = false) => {
+  const summary = {
+    ledger_index: Number(ledger.ledger_index),
+    ledger_hash: ledger.ledger_hash,
+    parent_hash: ledger.parent_hash,
+    close_time: convertRippleDate(ledger.close_time),
+    total_xrp: ledger.total_coins / 1000000,
```

Comment:
Might be good to define a constant for `100000` to avoid any off-by-one-zero typos.

Reply:
turns out the constant already exists and just wasn't used lol